homeassistant/components/ai_speaker/media_browser.py

```diff
@@ -0,0 +1,479 @@
+"""Support to interface with the AIS API."""
```

Comment:
Please start with a single platform in the first PR. Just keep the sensor platform here for now.

Reply:
ok, done in commit @asystentka-jolka
improvements after first review
6214ef8